getting-started/kubernetes/hardway/overview.md

```diff
@@ -4,16 +4,17 @@ description: A tutorial for installing Calico the hard way.
 canonical_url: '/getting-started/kubernetes/hardway/index'
 ---
 
-This tutorial walks you through setting up {{site.prodname}} the hard way.
+### About this tutorial
 
- - If you are looking to get up and running quickly with {{site.prodname}}, to try things out, check out our [quickstart guide]({{site.baseurl}}/getting-started/kubernetes/quickstart).
- - If you are looking for the most direct path to a production-ready {{site.prodname}} install, check out our [install guides]({{site.baseurl}}/getting-started/kubernetes/self-managed-onprem/onpremises).
-
-{{site.prodname}} the hard way is optimized for learning about how {{site.prodname}} works and what the other guides do “under the hood.”
+The "Hard Way" is a tutorial for learning how Kubernetes and {{site.prodname}} components fit together. It is a manifest-based install, and is optimized for learning what happens “under the hood.” **It is not intended for production installs** (which is an operator-based).
```

Comment:
Mixed ascii and unicode quotes in this para.

"which is an operator based": "which" doesn't refer back to anything, and, "an operator-based _what_"? 

In open source, the manifest-based installs are production ready too; just not this particular one. (In addition, the operator is not rolled out to all scenarios yet.)

Reply:
Argh! Rewrite includes your suggestions.